### DISCUSSION TITLE
[Help] Filtering notes based on YAML entries

### DISCUSSION THREAD

**Discussion by asterixpound** [zk-org/zk#356] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

I would like to be able to filter notes based on entries in the YAML frontmatter but I am not seeing a way of doing this in the docs on [note filtering](https://github.com/mickael-menu/zk/blob/main/docs/note-filtering.md).

Does zk support this?

**Comment by mickael-menu** [comment, marked as answer]
No, it's not possible. I considered doing something like this but never got around to it: https://github.com/mickael-menu/zk/issues/1#issuecomment-1001203350

**Reply by asterixpound** [reply]
Thanks for your reply @mickael-menu. It is a feature that would be useful to me if it ever becomes available. For now, I think I will get around this by moving some of the entries I store as separate YAML entries to tags.
